openmetadata-ui/src/main/resources/ui/src/components/TableTags/TableTags.component.tsx

```diff
@@ -50,34 +53,67 @@ const TableTags = ({
   fetchTags,
   tagFetchFailed,
   dataTestId,
-  placeholder,
-}: TableTagsComponentProps) => {
+}: TableTagsComponentProps<T>) => {
   const { t } = useTranslation();
-  const [editColumnTag, setEditColumnTag] = useState<{
-    column: Column;
-    index: number;
-  }>();
+  const [editColumnTag, setEditColumnTag] = useState<EditColumnTag<T>>();
 
   const columnFieldFQN = useMemo(
     () =>
-      `${EntityField.COLUMNS}${ENTITY_LINK_SEPARATOR}${getColumnName(
+      `${EntityField.COLUMNS}${ENTITY_LINK_SEPARATOR}${getColumnName?.(
```

Comment:
get columnaName into separate variable first, If that method was not found it will result in `undefined` within the string

Reply:
@chirag-madlani  This is used for requests and updates for tags in the table column, which is only supported by `Table Entity`.